modules/articles/client/config/articles.client.routes.js

```diff
@@ -12,22 +12,57 @@ angular.module('articles').config(['$stateProvider',
       })
       .state('articles.list', {
         url: '',
-        templateUrl: 'modules/articles/client/views/list-articles.client.view.html'
+        templateUrl: 'modules/articles/client/views/list-articles.client.view.html',
+        controller: 'ArticlesListController',
+        resolve:{
+          articles: function (Articles, $stateParams, $state) {
```

Comment:
Why not?  Do we not want the most recent data on the route resolve?  Not sure why we would want to call it in the controller after moving everything else to resolve.